README.md

```
@@ -1,2 +1,198 @@
+> `@testing-library/react` 기반으로 UI 컴포넌트 테스트를 수행합니다.
+
+---
+
+### 5️⃣ Storybook 실행
+
+```bash
+# Storybook 개발 서버 실행
+yarn storybook
+```
+
+> 기본 포트: **http://localhost:6006**
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)